src/components/Cashout/Components/Initial.view.tsx

```diff
@@ -40,11 +40,10 @@ export const InitialCashoutView = ({
     const { user, fetchUser, isFetchingUser } = useAuth()
     const [, setUserType] = useState<'NEW' | 'EXISTING' | undefined>(undefined)
 
-    const xchainAllowed = useMemo(
-        () =>
-            crossChainDetails.find((chain: any) => chain.chainId.toString() === selectedChainID.toString()) ||
-            selectedChainID === '1',
-        [crossChainDetails, selectedChainID]
+    const xchainAllowed = useMemo( (): boolean => {
+        return crossChainDetails.find((chain: any) => chain.chainId.toString() === selectedChainID.toString()) != undefined
+        },
+        [selectedChainID]
```

Comment:
suggestion: would be good to have  a comment explaining what this code actually does

Reply:
(ie: checking if we support x-chain on that chain)